File: ClientApp/src/app/clases/curriculumconvocatoria.ts
export class curriculumconvocatoria{

    idcurriculum : number;
    idconvocatoria : number;
    puntaje: number;
    prioridad: number;


    constructor(obj:any){
        //el orden de os parametros afecta la vista de los input en el modal

        this.idcurriculum= +obj.idcurriculum;
        this.idconvocatoria= +obj.idconvocatoria;
        this.puntaje= +obj.puntaje;
        this.prioridad= +obj.prioridad;
         // formateo a number el string // parseInt(val

    }

}

File: modelo/curriculumconvocatoria.cs
using System.Data;
using System;
    public class curriculumconvocatoria : oObjeto
    {
        public int Idcurriculum { get; set; }
        public int Idconvocatoria { get; set; }
        public int Puntaje{set;get;}
        public int Prioridad{get;set;}
        public curriculumconvocatoria(){}
        public curriculumconvocatoria(DataRow dr)
    {
        this.Idcurriculum = Convert.ToInt32(dr["idcurriculum"]);
        this.Idconvocatoria= Convert.ToInt32(dr["idconvocatoria"]);
        this.Puntaje= Convert.ToInt32(dr["puntaje"]);
        this.Prioridad=Convert.ToInt32(dr["prioridad"]);
        
    }

    }
 